docs/data/sources/google-tag-manager-client.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Google Tag Manager Template - Amplitude Analytics Browser SDK
+description: Collect data with ease using Amplitude Analytics Browser SDK GTM template - the official client-side Google Tag Manager template for seamless data collection.
+---
+
+This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Amplitude Marketing Analytics SDK](../../sdks/marketing-analytics-browser/) for data collection.
+
+!!!info Resources
+    [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-browser-sdk-gtm-template)
+
+With this template, you can defer the `init` tag until you receive a signal, like a consent grant. But make sure to fire the `init` tag before any other tags. The legacy template automatically calls init, so there's no need to do it yourself.
```

Comment:
```suggestion
An `init` tag is required to enable tracking. It is common to set up the `init` tag to be triggered by "Page Load" event. However, you can defer the `init` tag until you receive a signal, like user consent. Tracking is held off until `init` tag is triggered.
```

Reply:
Casey's pro tip: It is recommended to avoid using passive voice cause passive voice can obscure the agent of the action. 

Will add "It is common to set up the `init` tag to be triggered by "Page Load" event."